coverage.py

```diff
@@ -919,6 +921,17 @@ def _default_replacement_policy(self, _db):
         """
         return ReplacementPolicy.from_license_source(_db)
 
+    @classmethod
+    def collections(cls, _db):
```

Comment:
Pulled out of `CollectionCoverageProvider.all` below, to support any reordering work done by subclasses (in this case `IdentifierResolutionCoverageProvider`).